package.json

```diff
@@ -389,6 +389,11 @@
                     "azureFunctions.deploySubpath": {
                         "type": "string",
                         "description": "%azFunc.deploySubpathDescription%"
+                    },
+                    "azureFunctions.checkFunctionRuntime": {
```

Comment:
Suggestion: "azureFunctions.showCoreToolsWarning" (fyi, a lot of these suggestions are to be consistent with our other extensions, like [here](https://github.com/Microsoft/vscode-azureappservice/blob/master/package.json#L496))